modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,249 @@
+      --must-gather-dir-path string       Must gather directory path (default "must-gather")
+      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
+      --profile-name string               Name of the performance profile to be created (default "performance")
+      --reserved-cpu-count int            Number of reserved CPUs (required)
+      --rt-kernel                         Enable Real Time Kernel (required) (default true)
+      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
+      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
+      --user-level-networking             Run with User level Networking(DPDK) enabled
+----
++
+[NOTE]
+====
+There two types of arguments, wrapper arguments (help, PAO image and the tarball path) and the PPC arguments.
+====
++
+<1> Specify the Performance Addon Operator image, this field is optional and if nothing is set the default upstream image is used namely `quay.io/openshift-kni/performance-addon-operator:4.8-snapshot`.
+<2> `-t` is a required wrapper script argument and specifies the path to a `must-gather` tarball.
+
+. Check the machine config pool:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
+
+[source,terminal]
+----
+NAME         CONFIG                                                 UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master       rendered-master-acd1358917e9f98cbdb599aea622d78b       True      False      False      3              3                   3                     0                      22h
+worker-cnf   rendered-worker-cnf-1d871ac76e1951d32b2fe92369879826   False     True       False      2              1                   1                     0                      22h
+----
+
+. Create a performance profile:
++
+[source,terminal]
+----
+$ ./run-perf-profile-creator.sh -t /must-gather/must-gather.tar.gz -- --mcp-name=worker-cnf --reserved-cpu-count=2 --rt-kernel=true > my-performance-profile.yaml
+----
++
+[NOTE]
+====
+The Performance Profile Creator arguments are shown in the Performance Profile Creator arguments table. The following arguments are required:
+
+* `reserved-cpu-count`
+* `mcp-name`
+* `rt-kernel`
+
+The `mcp-name` in this example is set to `worker-cnf` based on the output of the command `oc get mcp`.
+====
+
+. Review the created YAML file:
++
+[source,terminal]
+----
+$ cat my-performance-profile.yaml
+----
+.Example output
++
+[source,terminal]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+    cpu:
+    isolated: 1-39,41-79
+    reserved: 0-49
```

Comment:
Minor fix here it should be: reserved: 0,40  (not 0-49, This will mean 50 reserved cpus)